lib/mayaUsd/fileio/registryHelper.cpp

```diff
@@ -131,10 +131,10 @@ static bool _HasShadingModePlugin(
 
     JsValue any;
     if (TfMapLookup(mayaTranslatorMetadata, _tokens->mayaPlugin, &any)) {
-        return _GetData(any, mayaPluginName);
+        _GetData(any, mayaPluginName);
     }
 
-    return false;
+    return true;
```

Comment:
Return true as soon as we found the Info:UsdMaya:ShadingModePlugin. The mayaPlugin resolution becomes optional.

Reply:
Documentation would be very welcome here. Things that will let developers know that mayaPluginName is a return argument, that it may not be initialized (if someone forgot to do so), and that shading mode plugin doesn't have to be associated with maya plugin.